src/main/java/org/apache/sling/maven/feature/launcher/StartMojo.java

```diff
@@ -124,24 +171,58 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 File featureFile = result.getArtifact().getFile();
                 
                 List<String> args = new ArrayList<>();
-                String javahome = System.getenv("JAVA_HOME");
-                if (javahome == null || javahome.isEmpty()) {
-                    // SLING-9843 fallback to java.home system property if JAVA_HOME env variable is not set
-                    getLog().warn("The JAVA_HOME env variable was not set, falling back to the java.home system property");
-                    javahome = System.getProperty("java.home");
-                }
-                args.add(javahome + File.separatorChar + "bin" + File.separatorChar + "java");
-                // SLING-9994 - if any extra vm options were supplied, apply them here
-                String[] vmOptions = launch.getLauncherArguments().getVmOptions();
-                if (vmOptions != null) {
-                    for (String vmOption : vmOptions) {
-                        if (vmOption != null && !vmOption.isEmpty()) {
-                            args.add(vmOption);
+                if (useAssembly) {
```

Comment:
I have not checked, but I assume that the launcher script consults the `JAVA_HOME` env var, right?

Reply:
Yes the launcher script does have some logic for finding where the java executable is.  But thinking more about this more, it probably wouldn't hurt to set the JAVA_HOME env variable here to deal with things like SLING-9843

I''ll make that change.